### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by horsemaker**

### CODE REVIEW

File: src/frontend/components/ProfileSection.js [hunk 1]
```diff
@@ -126,11 +168,23 @@ const ProfileSection = ({ profile }) => {
...
+        onClose={onCloseFollowing}
+        title={"Followering"}
+        list={following}
+      />
+      <ShowListModal
+        isOpen={isOpenFollowers}
+        onClose={onCloseFollowers}
+        title={"Followers"}
+        list={followers}
+      />
```
Comment (narharikale) [inline src/frontend/components/ProfileSection.js]: loved the likes modal 💯

**Review by narharikale — APPROVED** [review]
great work yash 🚀

File: src/frontend/components/ProfileSection.js [hunk 2]
```diff
@@ -126,11 +168,23 @@ const ProfileSection = ({ profile }) => {
...
-        initialRef={initialRef}
+        isOpen={isOpenEditProfile}
+        onClose={onCloseEditProfile}
+        initialRef={initialRefEditProfile}
         profile={profile}
       />
+      <ShowListModal
+        isOpen={isOpenFollowing}
+        onClose={onCloseFollowing}
+        title={"Followering"}
```
Comment (MarkVed17) [inline src/frontend/components/ProfileSection.js]: typo here

**Review by MarkVed17 — APPROVED** [review]
Looks good!
You may proceed.

**Pull request merged**